packages/jellyfish-api-core/__tests__/category/account/getAccountHistory.test.ts

```diff
@@ -0,0 +1,84 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import waitForExpect from 'wait-for-expect'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+function createTokenForContainer (container: MasterNodeRegTestContainer) {
+  return async (address: string, symbol: string, amount: number) => {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+}
+
+describe('Account', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+  const createToken = createTokenForContainer(testing.container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+    await testing.container.waitForWalletBalanceGTE(100)
+    await createToken(await testing.container.getNewAddress(), 'DBTC', 200)
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should getAccountHistory', async () => {
+    await waitForExpect(async () => {
+      const accountHistories = await testing.rpc.account.listAccountHistory()
+      expect(accountHistories.length).toBeGreaterThan(0)
+    })
+
+    const accountHistories = await testing.rpc.account.listAccountHistory()
+    const referenceHistory = accountHistories[0]
+
+    const history = await testing.rpc.account.getAccountHistory(referenceHistory.owner, referenceHistory.blockHeight, referenceHistory.txn)
+    expect(history.owner).toStrictEqual(referenceHistory.owner)
+    expect(history.blockHeight).toStrictEqual(referenceHistory.blockHeight)
+    expect(history.txn).toStrictEqual(referenceHistory.txn)
+    expect(history.txid).toStrictEqual(referenceHistory.txid)
+    expect(history.type).toStrictEqual(referenceHistory.type)
+  })
+
+  it('should not getAccountHistory', async () => {
```

Comment:
personally i think it would be clearer to seperate the failing tests out to their own individual tests as it gives me clarity as to what would fail

eg. 
- should not getAccountHistory when address is not valid
....